protocol/git/README.md

```diff
@@ -47,8 +47,9 @@ Write a [good commit message]. Example format:
 
     http://project.management-system.com/ticket/123
 
-If you've created more than one commit, use a rebase to squash them into
-cohesive commits with good messages:
+If you've created more than one commit, use a
```

Comment:
What about linking a whole phrase rather than part of a phrase? Also "use a rebase" sounds awkward, what do you think about rephrasing this to "use `git rebase`" instead?

So all together it would be:

> [use `git rebase`](http://some-link-here.com) to squash them into cohesive commits with good messages